javascript-bank/bank.js

```diff
@@ -0,0 +1,38 @@
+/* exported Bank */
+function Bank() {
+  this.nextAccountNumber = 1;
+  this.accounts = [];
+}
+
+Bank.prototype.openAccount = function (holder, balance) {
+  if (Number.isInteger(balance) && balance > 0) {
+    var newAccount = new Account(this.nextAccountNumber, holder);
+    newAccount.deposit(balance);
+    this.accounts.push(newAccount);
+    this.nextAccountNumber++;
+    return newAccount.number;
+  } else {
+    return null;
+  }
+};
+
+Bank.prototype.getAccount = function (number) {
+  for (var i = 0; i < this.accounts.length; i++) {
+    if (this.accounts[i].number === number) {
+      return this.accounts[i];
+    }
+  }
+  return null;
+};
+
+Bank.prototype.getTotalAssets = function () {
+  var grandTotal = 0;
+  if (this.accounts.length === 0) {
+    return 0;
+  }
```

Comment:
This isn't necessary. If the array is empty, the loop would never start, and you would return `0` automatically as the default value for the total